src/neo-vm/ExecutionEngine.cs

```diff
@@ -126,866 +126,874 @@ private bool ExecuteInstruction()
-                        {
-                            ExecutionContext context_pop = InvocationStack.Pop();
-                            int rvcount = context_pop.RVCount;
-                            if (rvcount == -1) rvcount = context_pop.EvaluationStack.Count;
-                            EvaluationStack stack_eval;
-                            if (InvocationStack.Count == 0)
-                            {
-                                EntryContext = null;
-                                CurrentContext = null;
-                                stack_eval = ResultStack;
-                            }
-                            else
-                            {
-                                CurrentContext = InvocationStack.Peek();
-                                stack_eval = CurrentContext.EvaluationStack;
-                            }
-                            if (context_pop.EvaluationStack == stack_eval)
-                            {
-                                if (context_pop.RVCount != 0) return false;
-                            }
-                            else
-                            {
-                                if (context_pop.EvaluationStack.Count != rvcount) return false;
-                                if (rvcount > 0)
-                                    context_pop.EvaluationStack.CopyTo(stack_eval);
-                            }
-                            if (InvocationStack.Count == 0 || context_pop.AltStack != CurrentContext.AltStack)
-                            {
-                                context_pop.AltStack.Clear();
-                            }
-                            if (InvocationStack.Count == 0)
-                            {
-                                State = VMState.HALT;
-                            }
-                            ContextUnloaded(context_pop);
-                            return true;
-                        }
-                    case OpCode.SYSCALL:
-                        {
-                            if (!OnSysCall(instruction.TokenU32))
-                                return false;
-                            break;
-                        }
+                case OpCode.PUSHINT8:
+                case OpCode.PUSHINT16:
+                case OpCode.PUSHINT32:
+                case OpCode.PUSHINT64:
+                case OpCode.PUSHINT128:
+                case OpCode.PUSHINT256:
+                    {
+                        Push(new BigInteger(instruction.Operand.Span));
+                        break;
+                    }
+                case OpCode.PUSHNULL:
+                    {
+                        Push(StackItem.Null);
+                        break;
+                    }
+                case OpCode.PUSHDATA1:
+                case OpCode.PUSHDATA2:
+                case OpCode.PUSHDATA4:
+                    {
+                        if (!CheckMaxItemSize(instruction.Operand.Length)) return false;
```

Comment:
I think that this line could be removed because the max length will be 4

Reply:
The max length will be `(1 << (4 * 8)) - 1`.

Reply:
I will review it again

Reply:
It's good

Reply:
But the max length is 1Mb is not too much?

Reply:
For hit this line I need a two mb json xD

Reply:
Then we can temporarily not cover this line.